src/test/regress/expected/fkeys_between_local_ref.out

```diff
@@ -99,10 +99,6 @@ CREATE TABLE partitioned_table_1_100_200 PARTITION OF partitioned_table_1 FOR VA
 CREATE TABLE partitioned_table_1_200_300 PARTITION OF partitioned_table_1 FOR VALUES FROM (200) TO (300);
 INSERT INTO partitioned_table_1 SELECT i FROM generate_series(195, 205) i;
 ALTER TABLE partitioned_table_1 ADD CONSTRAINT fkey_8 FOREIGN KEY (col_1) REFERENCES local_table_4(col_1);
--- now that we attached partitioned table to graph below errors out
```

Comment:
I think instead of removing these tests in this file, we should keep because the tables are inside pretty useful/complex foreign key graphs 

(also, updating the comments)